parts/kubernetesmaster-kube-apiserver.yaml

```diff
@@ -29,6 +29,9 @@ spec:
         - "--tls-private-key-file=/etc/kubernetes/certs/apiserver.key"
         - "--client-ca-file=/etc/kubernetes/certs/ca.crt"
         - "--service-account-key-file=/etc/kubernetes/certs/apiserver.key"
+        - "--oidc-client-id=<oidcClientId>"
+        - "--oidc-issuer-url=<oidcIssuerUrl>"
+        - "--oidc-username-claim=oid"
```

Comment:
@lachie83 
`sub` is recommended, it is bound to both user identity and the auth request, but does not have a directly mapped entry in AAD

`email` and `upn`, those are usually user email addresses, but not guaranteed to be available for every user. Based on a recent test:
For organizational users, they have 'upn', but not 'email'.
For Microsoft Account external users, they have 'email', but not 'upn'.
Service principal users have neither of them.

`oid` is the one present for all kinds of users, and have a directly mapped entry in AAD.

Refer to:
https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-code